.github/workflows/backup-kubernetes-databases-inhouse.yml

```diff
@@ -0,0 +1,168 @@
+name: Backup kubernetes postgres databases.
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'Define the type of machine to run the job on'
+        type: string
+        required: false
+        default: '["self-hosted", "Linux", "X64", "bratislava"]'
+      backup:
+        description: 'Two types of backups postgress or shared-wordpress'
+        type: string
+        required: false
+        default: 'postgres'
+      cluster:
+        description: 'Kubernetes cluster name'
+        required: true
+        type: string
+      namespace:
+        description: 'Namespace where should be build app deployed'
```

Comment:
I guess this a remnant from coping from other pipeline, as we aren't deploying any app